packages/webcert/src/feature/certificate/UnifiedEdit/UeMedicalInvestigation/UeMedicalInvestigation.tsx

```diff
@@ -51,12 +51,16 @@ const UeMedicalInvestigation: React.FC<Props> = ({ disabled, config, value, vali
   }
 
   const handleInformationSourceChange = (text: string) => {
+    const informationSource = { ...value.informationSource }
```

Comment:
Test should be added to verify that the value is set to null if the text is empty and that the value is not set to null if the text is not empty.